backend/src/main/java/com/redhat/cloud/notifications/processors/email/EmailAggregator.java

```diff
@@ -60,42 +57,38 @@
                 .onItem().transformToMulti(Multi.createFrom()::iterable)
                 // For each aggregation...
                 .onItem().transformToUniAndConcatenate(aggregation -> {
-                    // We need to parse the action to determine the target endpoints.
-                    return actionParser.fromJsonString(aggregation.getPayload().encode())
```

Comment:
It was failing here because our aggregation payload is not a valid `Action` nor do we need it to be a valid (at least not for now). 
It's build by the aggregation code and is not encoded to `Action`, so the payload and context are actually json objects and not json strings.